src/customizations/components/manage/Blocks/Listing/ListingBody.jsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
```

Comment:
c'è un problema se al blocco si mette lo sfondo: la paginazione rimane fuori dallo sfondo.
Siccome la classe per applicare lo sfondo è applicata nel singolo template, in realtà per fare in modo che includa anche la paginazione, andrebbe messa in questo componente e tolta dai vari template. Per capire se ci deve essere lo sfondo, secondo me lo si riesce a vedere da data.block. 
La classe per lo sfondo è bg-light.
Attenzione che il template 'Gallery a griglia' ha classi diverse per lo sfondo: non usa bg-light ma "section section-muted section-inset-shadow" e deve rimanere così